Week2/MohammadDarwesh/mohammaddarwesh.js

```diff
@@ -0,0 +1,180 @@
+// for the first task i will do it with many ways just for practising.
```

Comment:
Hi Mohammad, the homework also contains the code from week2, next time you should make a new branch from `master` so only the code for week3 is in the pull request.